io.github.aliasbody.xenia.yml

```diff
@@ -0,0 +1,96 @@
+id: io.github.aliasbody.flatpak-xenia
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: xenia.sh
+
+# Support 32-bit at runtime
+add-extensions:
+  org.freedesktop.Platform.Compat.i386:
+    directory: lib/i386-linux-gnu
+    version: '23.08'
+    
+  org.freedesktop.Platform.GL32:
+    directory: lib/i386-linux-gnu/GL
+    versions: "23.08;23.3.1"
+    version: '23.3.1'
+    subdirectories: true
+    no-autodownload: true
+    autodelete: false
+    add-ld-path: lib
+    merge-dirs: vulkan/icd.d;glvnd/egl_vendor.d;OpenCL/vendors;lib/dri;lib/d3d;vulkan/explicit_layer.d
+    download-if: active-gl-driver
+    enable-if: active-gl-driver
+    
+    org.freedesktop.Platform.ffmpeg-full:
+      directory: lib/ffmpeg
+      add-ld-path: .
+      version: '23.08'
+      autodelete: false
+
+  org.freedesktop.Platform.ffmpeg_full.i386:
+      directory: lib32/ffmpeg
+      add-ld-path: .
+      version: '23.08'
+      autodelete: false
+
+# Support 32-bit at buildtime
+sdk-extensions:
+  - org.freedesktop.Sdk.Compat.i386
+  
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

Comment:
Does it suppoort native wayland? If yes use fallback-x11 + wayland otherwise x11 only.